src/webextension/widgets/panel.js

```diff
@@ -12,12 +13,15 @@ import styles from "./panel.css";
 export function PanelHeader({className, onBack, children}) {
   const finalClassName = `${styles.panelHeader} ${className}`.trimRight();
   const imgSrc = browser.extension.getURL("/icons/arrowhead-left-16.svg");
-  // TODO: actually localize this string
-  const imgAlt = "gO bACk";
   return (
     <header className={finalClassName}>
-      {onBack ? <Button theme="ghost" size="micro" onClick={onBack}>
-        <img src={imgSrc} alt={imgAlt}/></Button> : null}
+      {onBack ? (
+        <Button theme="ghost" size="micro" onClick={onBack}>
+          <Localized id="panel-back-button">
```

Comment:
I swear I tried this (amongst a few other things). I really get confused on how this Localized component works sometimes... thanks for this.